conversions/octal_to_binary.py

```diff
@@ -0,0 +1,48 @@
+"""
+* Author: Bama Charan Chhandogi (https://github.com/BamaCharanChhandogi)
+* Description: Convert a Octal number to Binary.
+
+reference for better understand
+
+URL: https://en.wikipedia.org/wiki/Binary_number
+URL: https://en.wikipedia.org/wiki/Octal
+"""
+
+
+def octal_to_binary(octal_number: str) -> str:
+    binary_number = ""
+    octal_digits = "01234567"
+
+    """
+    ValueError: String to the function
```

Comment:
add doctests for special character input, empty input.

Reply:
Check!